src/exampleTypes.js

```diff
@@ -1,11 +1,15 @@
-import { defaultHasValue } from './validation'
-// import {toSentence} from 'underscore.string.fp'
+import _ from 'lodash/fp'
+import * as F from 'futil-js'
+
+let checkValueOn = (...fields) => _.flow(F.cascade(fields), _.negate(_.isEmpty))
+let validate = checkValueOn('data.values', 'values', 'value')
 
 export default {
   default: {
-    validate: defaultHasValue,
+    validate: x => true,
```

Comment:
you can kill the default type here completely because runTypeFunction will default to returning true